tools/src/main/scala/scala/scalanative/build/ScalaNative.scala

```diff
@@ -87,12 +87,11 @@ private[scalanative] object ScalaNative {
 
   /** Given low-level assembly, emit LLVM IR for it to the buildDirectory. */
   def codegen(config: Config, linked: linker.Result): Seq[Path] = {
-    config.logger.time("Generating intermediate code") {
+    val llPaths = config.logger.time("Generating intermediate code") {
       CodeGen(config, linked)
     }
-    val produced = IO.getAll(config.workdir, "glob:**.ll")
-    config.logger.info(s"Produced ${produced.length} files")
-    produced
+    config.logger.info(s"Produced ${llPaths.length} files")
```

Comment:
let add a `target triple` here or use generic line if none.

Reply:
This is the number of LL files and the strategies are described in `GenCode`. The very next step in compilation prints the target triple so I think this is fine.